spec/api/v1/users_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe V1::Users do
+  let(:api_token) { create :api_token }
+
+  describe '#GET /api/v1/users' do
+    context 'when api token is invalid' do
+      let(:api_token) { double :api_token, token: 'foo' }
+
+      subject { get '/api/v1/users', api_key: api_token.token }
```

Comment:
What do you think of using `before` instead of `subject`? Then you don't need to call `subject` in `it`.